package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "nodule-openapi",
+    "version": "0.1.0",
+    "description": "Opinionated OpenAPI (Swagger) Client",
+    "main": "src/index.js",
```

Comment:
It's far more typical for a package's `main` field to point to a `lib/` folder where the transpiled code goes into

Reply:
Just `lib/` or `lib/index.js` or what?

Reply:
so long as there is an index.js file in lib/ you can just keep it as `lib/`

Reply:
lib/index.js